### ISSUE TITLE
TO DO

### ISSUE THREAD

**Issue by Protohedgehog** [OpenScienceMOOC/Module-6-Open-Access-to-Research-Papers#10]
Stuff to add in https://psyarxiv.com/e9zs8

NOTE: ADD STUFF IN; DELETE WHEN ADDED

**Comment by Protohedgehog** [comment]
https://www.biorxiv.org/content/10.1101/673665v1

**Comment by Protohedgehog** [comment]
https://bmjopen.bmj.com/content/9/6/e028655.info

**Comment by Protohedgehog** [comment]
~~For OA in HSS: https://training.parthenos-project.eu/ 
https://genr.eu/wp/humanities-progressing-in-open-science/
https://dariahopen.hypotheses.org/
https://digitalscience.figshare.com/articles/The_State_of_Open_Monographs/8197625
https://www.cambridge.org/core/books/open-access-and-the-humanities/02BD7DB4A5172A864C432DBFD86E5FB4
https://dariahopen.hypotheses.org/529
https://training.parthenos-project.eu/sample-page/citizen-science-in-the-digital-arts-and-humanities/
https://dariahopen.hypotheses.org/6~~

EDIT 20190805: incorporated into file `13_MAIN_Open_Access_monographs_and_books.md`

**Comment by tosteiner** [comment]
On CC licensing: https://m.youtube.com/watch?feature=youtu.be&v=oLcVycFkmtM

**Comment by Protohedgehog** [comment]
Nice, thanks @tosteiner. I thought we had a video on CC licenses already in this module, but I can't seem to find it now.. -_-

**Comment by tosteiner** [comment]
Yeah, I've lost count of the vids I encountered that do a decent job at explaining how the CC system works :) The one from Cable (C Green, above) is brand new, so I thought this might fit in nicely :) 

Also, still love this one on OA, which has a small bit on how CC licenses work: https://youtu.be/gzRgknylTEM 

EDIT: on second note, the one mentioned [here](https://github.com/OpenScienceMOOC/Module-6-Open-Access-to-Research-Papers/issues/10#issuecomment-515797158) would fit the OER module even better ... ;)

**Comment by Protohedgehog** [comment]
https://rss.onlinelibrary.wiley.com/doi/abs/10.1111/j.1740-9713.2019.01299.x

**Comment by Protohedgehog** [comment]
https://journals.sagepub.com/doi/abs/10.1177/1745691619858427

**Comment by Protohedgehog** [comment]
https://journals.sagepub.com/doi/abs/10.1177/1745691619858427
https://jpma.org.pk/article-details/9322?article_id=9322

**Comment by Protohedgehog** [comment]
Add in https://www.biorxiv.org/content/10.1101/795310v1
